WcaOnRails/db/migrate/20170816143703_add_greater_china_championship_type.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddGreaterChinaChampionshipType < ActiveRecord::Migration[5.1]
+  def change
+    %w(CN HK MO TW).each do |iso2|
+      EligibleCountryIso2ForChampionship.create!(championship_type: "greater_china", eligible_country_iso2: iso2)
```

Comment:
Yeah, the WCA Board is going to have to ask the WST to create new championship types as needed. I think this will be fine, as I expect the rate of requests to be exceedingly low.

It's also worth noting that the people creating competitions don't even have the power to set the championship type of a competition, they need to ask the Board to set it for them.